src/main/kotlin/com/zepben/evolve/streaming/get/NetworkConsumerClient.kt

```diff
@@ -80,8 +80,11 @@ class NetworkConsumerClient(
      * - When [GrpcResult.wasFailure], the error that occurred retrieving or processing the the object, accessible via [GrpcResult.thrown].
      * Note the [NetworkConsumerClient] warning in this case.
      */
-    fun getEquipmentForContainer(equipmentContainer: EquipmentContainer): GrpcResult<MultiObjectResult> =
-        getEquipmentForContainer(equipmentContainer.mRID)
+    fun getEquipmentForContainer(
```

Comment:
todo: Need to add @JvmOverloads to public functions with default arguments to keep Java compatibility. This applied to all functions that have been updated.